dependencies.gradle

```diff
@@ -86,7 +86,7 @@ ext {
         kotshiApi              : "se.ansman.kotshi:api:${versions.kotshi}",
         kotshiCompiler         : "se.ansman.kotshi:compiler:${versions.kotshi}",
 
-        frescoImageViewer      : "com.github.stfalcon:frescoimageviewer:${versions.frescoImageViewer}",
+        frescoImageViewer      : "com.github.luciofm:FrescoImageViewer:${versions.frescoImageViewer}",
```

Comment:
Uhmm. I don't trust very much on libraries coming from that github... :laughing:

Reply:
Just waiting Picasso3 to get rid of it 😄